src/array.jl

```diff
@@ -16,6 +16,8 @@ end
 
 CuArray{T,N}(buf::Mem.Buffer, dims::NTuple{N,Integer}) where {T,N} = CuArray{T,N}(buf, 0, dims)
 
+Base.pointer(A::CuArray) = unsafe_buffer(A)
```

Comment:
Do we need this?

Reply:
I thought I relied on that - but I need to assure.